devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1029,6 +1239,60 @@ bool WholeBodyDynamicsDevice::attachAllFTs(const PolyDriverList& p)
         return false;
     }
 
+    // Attach the controlBoardList to the controlBoardRemapper
+    bool ok = remappedMASInterfaces.multwrap->attachAll(ftSensorList);
+    ok = ok & remappedMASInterfaces.multwrap->attachAll(tempSensorList);
+
+    if( !ok  )
+    {
+        yError() << " WholeBodyDynamicsDevice::attachAll in attachAll of the remappedMASInterfaces";
+        return false;
+    }
+
+    // Check if the MASremapper and the estimator have a consistent number of ft sensors
+    int tempSensors = 0;
+    tempSensors=static_cast<int>( remappedMASInterfaces.temperatureSensors->getNrOfTemperatureSensors());
+    if( tempSensors > static_cast<int>( estimator.sensors().getNrOfSensors(iDynTree::SIX_AXIS_FORCE_TORQUE)) )
+    {
+        yError() << "wholeBodyDynamics : The multipleAnalogRemappedDevice has more sensors than those in the open estimator ft sensors list";
+        return false;
+    }
+
+    // Temporary code to check name of sensors, assuming the temperature sensor name will be the same as the ft sensor
```

Comment:
Is this temporary code necessary or, can it be removed?

Reply:
Well is what was agreed when developing the logic it needs to set the proper names in the configuration files of the sensor. So it is not really temporary anymore, removed this line.
The problem is I do not know where to document this at the moment.